che-theia-java-extension/src/browser/che-theia-java-frontend-module.ts

```diff
@@ -13,19 +13,25 @@
 import { JavaExtensionContribution } from './che-theia-java-contribution';
 import {
     CommandContribution,
-    MenuContribution
+    MenuContribution,
+    ResourceResolver
 } from "@theia/core/lib/common";
 
 import { ContainerModule } from "inversify";
-import { KeybindingContribution, KeybindingContext } from '@theia/core/lib/browser';
+import { KeybindingContribution, KeybindingContext, WidgetFactory} from '@theia/core/lib/browser';
 
-import "../../src/browser/styles/icons.css";
 import { FileStructure } from './navigation/file-structure';
-import { FindImplementers } from './navigation/find-implementers';
 import { JavaEditorTextFocusContext } from './java-keybinding-contexts';
 
-export default new ContainerModule((bind) => {
+import { ExternalLibrariesWidget, EXTERNAL_LIBRARIES_ID } from './libraries/external-libraries-widget';
+import { createExternalLibrariesWidget } from './libraries/external-libraries-container';
+import { CheLibResourceResolver } from './libraries/chelib-resource-provider';
+import { FileNavigatorWidget } from '@theia/navigator/lib/browser';
+
+import "../../src/browser/styles/icons.css";
```

Comment:
AFAIK there is a rule in upstream theia to use always `'` (here in the whole PR I see that sometimes `"` is used, sometimes `'`)

you may introduce linter such https://github.com/eclipse/che-theia-factory-extension/blob/master/tslint.json to enforce some rules